src/main/java/uk/co/evoco/webdriver/utils/SendKeysUtils.java

```diff
@@ -1,19 +1,23 @@
 package uk.co.evoco.webdriver.utils;
 
 import org.openqa.selenium.WebElement;
+import uk.co.evoco.webdriver.configuration.TestConfigManager;
 
 import java.util.Optional;
 
 public final class SendKeysUtils extends TolerantInteraction {
 
     /**
-     *
```

Comment:
These need adding back in - javadocs convention is to have a blank line first which is where the description goes